css/layout/footer.css

```
@@ -1,254 +1,253 @@
 .license-info p {
-	font-size: var(--font-size-sm);
-	font-weight: 400;
-	color: var(--color-text-light);
+  font-size: var(--font-size-sm);
+  font-weight: 400;
+  color: var(--color-text-light);
 
-	min-height: 20px;
-	display: flex;
-	align-items: center;
+  min-height: 20px;
```

Stylelint (17.2.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)